src/vm/ilmarshalers.h

```diff
@@ -1417,7 +1474,16 @@ class ILMarshaler
 
     void EmitInterfaceClearNative(ILCodeStream* pslILEmit);
 
+    virtual bool AlwaysConvertByValContentsCLRToNative()
+    {
+        LIMITED_METHOD_CONTRACT;
+        return false;
+    }
+
 public:
+    
+    // Extension point to allow a marshaler to conditionally override all of the ILMarshaler logic with its own or block marshalling when marshalling an argument.
+    // See mlinfo.cpp:3523 for the implementation.
```

Comment:
Like the comment - remove the line number and replace with function name.

Reply:
I think there are a few other spots this happened - i.e. line number.

Reply:
I only remember doing that on this line and again a few lines lower. I'll go verify now.

Reply:
Yeah those were the only two cases of that in this PR.